bash_completions.go

```diff
@@ -434,6 +438,16 @@ func writeRequiredNouns(buf *bytes.Buffer, cmd *Command) {
 	}
 }
 
+func writeCmdAliases(buf *bytes.Buffer, cmd *Command) {
+	sort.Sort(sort.StringSlice(cmd.Aliases))
+
+	for _, value := range cmd.Aliases {
+		buf.WriteString(fmt.Sprintf("    command_aliases+=(%q)\n", value))
+		buf.WriteString(fmt.Sprintf("    aliasHash[%q]=%q\n", value, cmd.Name()))
```

Comment:
I FINALLY looked at this. Since the (ancient) version of bash on OS X does not support associative arrays, we have the HORRIBLE contortions with `flags_with_completion` and trying to find the same index in 2 different arrays.

Do you know how this script will react on bash versions that don't support arrays? I see that flagHash does exist, but we never actually use it...  I think it could/should be ripped out of the code.

Should we do something like grab the bash version or remember if the declare -A worked, and just skip the words[c] rewrite if the version is too old?